tests/telemetry/test_telemetry.py

```diff
@@ -0,0 +1,47 @@
+import pytest
+import logging
+
+# Helper functions
+def get_dict_stdout(gnmi_out, certs_out):
+    result = ""
+    gnmi_list = []
+    gnmi_list = get_list_stdout(gnmi_out) + get_list_stdout(certs_out)
+    params_dict= dict(itertools.izip_longest(*[iter(gnmi_list)] *2, fillvalue=""))
```

Comment:
Add space before `=`

Also, can you explain exactly what you are achieving with this line?

Reply:
Will add space. Converting list (output form stdout)to the dictionary to resemble key, value structure

Reply:
I'm just curious if there is a simpler method of accomplishing this. Can you provide an example of a `gnmi_list` and a resulting `params_dict`?

Reply:
I believe `gnmi_list` contains a list like the following : `['key1', 'val1', 'key2', 'val2', 'key3', 'val3']` and you want to convert it such that `params_dict` contains the following: `{'key1': 'val1', 'key2': 'val2', 'key3': 'val3'}`. If this is the case, and you can guarantee that the list is always in this order and there is a value for every key, then the following code is more understandable:

```
gnmi_key_value_list = get_list_stdout(gnmi_out) + get_list_stdout(certs_out)
# Elements in list alternate between key and value. Separate them and combine into a dict.
gnmi_key_list = gnmi_key_value_list[0::2]
gnmi_value_list = gnmi_key_value_list[1::2]
params_dict = dict(zip(gnmi_key_list, gnmi_value_list))
```

Reply:
Yes when we set values in redis it is always going to be in key, value. So I cant think of a situation where we do not have value for a particular key in config_db. I will update accordingly. Thanks!

Reply:
OK. Please test it thoroughly to ensure I didn't make any assumptions or mistakes :)

Reply:
Tested. 
![image](https://user-images.githubusercontent.com/49077256/78859715-06fdf600-79e5-11ea-9e8b-2bafd8ac3fd6.png)

![image](https://user-images.githubusercontent.com/49077256/78859661-df0e9280-79e4-11ea-9f16-1107c0d895a5.png)